src/lib/apiTypesV2/Entity.cpp

```diff
@@ -38,6 +38,7 @@
 */
 Entity::Entity()
 {
+  typeGiven = false;
```

Comment:
Actually, the title of the PR should be "null type in parse step", right? I mean, the ability from the parser layer to distinguish between the situation in which the input payload includes `type: ""` (typeGiven = false) and absences of `type` (typeGiven = true).

Just to clarify this PR is not about the usage of `null` as value for an attribute, as boolean or numbers are.